inst/include/Rcpp/date_datetime/Date.h

```diff
@@ -102,6 +102,16 @@ namespace Rcpp {
             return m_d;
         }
 
+        inline std::string format(const char *fmt = "%Y-%m-%d") const {
+            char txt[32];
+            struct tm temp = m_tm;
+            temp.tm_year -= baseYear();    // adjust for fact that system has year rel. to 1900
+            ::strftime(txt, 31, fmt, &temp);
+            return std::string(txt);
```

Comment:
Per previous point and `man strftime`, by catching a return of zero we should catch that.